assets/dfn-panel.css

```diff
@@ -0,0 +1,60 @@
+/* dfn popup panel that list all local references to a dfn */
+dfn {
+  cursor: pointer;
+}
+.dfn-panel {
```

Comment:
Should we add a newline between each rule?

Reply:
Good point. For consistency.